Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -968,6 +970,40 @@ IPCCommandResult ES::GetConsumption(const IOCtlVRequest& request)
   return GetDefaultReply(IPC_SUCCESS);
 }
 
+static bool CanDeleteTitle(u64 title_id)
+{
+  // IOS only allows deleting non-system titles (or a system title higher than 1-101).
```

Comment:
Really? I thought it was 1 to 65 XvP. More seriously, the appropriate numbers are just below the comment. I personally don't mine any notation, even if having a standard hex prefix would make it clearer.